File: cpp/gabaritos/gabarito27.cpp
#include <iostream>
#include <cmath>

int main() {
    double x;
    std::cout << "Digite um número positivo: ";
    std::cin >> x;
    std::cout << "Raiz quadrada: " << std::sqrt(x) << std::endl;
    std::cout << "Seno: " << std::sin(x) << std::endl;
    std::cout << "Logaritmo natural: " << std::log(x) << std::endl;
    return 0;
}

File: csharp/gabaritos/gabarito27.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Digite um número positivo: ");
        double x = double.Parse(Console.ReadLine());
        Console.WriteLine($"Raiz quadrada: {Math.Sqrt(x)}");
        Console.WriteLine($"Seno: {Math.Sin(x)}");
        Console.WriteLine($"Logaritmo natural: {Math.Log(x)}");
    }
}